nss-tlsd.c

```diff
@@ -928,6 +930,14 @@ parse_cfg (const gboolean   root)
         return FALSE;
     }
 
+    if (root) {
+        g_free (username);
+        username = g_key_file_get_value (cfg, "global", "user", NULL);
```

Comment:
Why store it in a global?

Reply:
I didn’t want to change much, so I did the same thing it does with resolvers and command line arguments.